demystifying_machine_learning/demystifying_machine_learning.md

```diff
@@ -0,0 +1,712 @@
+
+For an example of anomaly detection in action, check out this article on [outlier detection for patient monitoring and alerting](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3567774/).
+The authors created a machine learning algorithm that was trained on the electronic health records of post-cardiac surgical patients with the intention of flagging any anomalies in patient-management decisions.
+By detecting decisions that deviate from typical patient care, the system can alert providers, potentially reducing medical errors.
+
+
+### Clustering
+
+The goal of clustering is to **group together observations** that show a similar pattern of response across variables.
+Clustering models are unsupervised, because there is no outcome variable present in the training data.
+
+For example, if you have a dataset of a variety of risk factors in adolescents, you may want to run a clustering analysis to see what typical profiles emerge (see [Walsh et al., 2020](https://doi.org/10.1016/j.jadohealth.2020.02.012)).
+You might see one group of adolescents who are more likely to report both substance use and early sex, and another group who report insufficient nutrition, and so on.
+
+Another common application of clustering algorithms is [identifying putative cell types in single-cell RNA sequencing](https://www.sciencedirect.com/science/article/pii/S0098299717300493?via%3Dihub).
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+Two common approaches for clustering analysis are [hierarchical clustering](https://en.wikipedia.org/wiki/Hierarchical_clustering) and [k-means clustering](https://en.wikipedia.org/wiki/K-means_clustering), but there are many more techniques available.
+
+For a general overview of clustering algorithms, see the [Google course on clustering](https://developers.google.com/machine-learning/clustering/clustering-algorithms).
+
+For an overview of clustering algorithms specifically as applied in single cell RNA sequencing, see [Duò, Robinson and Soneson, 2020](https://f1000research.com/articles/7-1141/v3).
+
+</div>
+
+### Dimension reduction
+
+The goal of dimension reduction is to **summarize complex data**, so it can be represented with **fewer variables** (fewer dimensions).
+For example, if you have a dataset with 100 variables, you might use dimension reduction to get that down to just 10 variables that still capture nearly all of the information from the original data.
+Like clustering, dimension reduction models are unsupervised.
+
+There are two main ways to reduce the number of variables in your data: drop variables that you don't need, or summarize a lot of variables with fewer variables.
+Dropping unnecessary variables is called [feature selection](https://en.wikipedia.org/wiki/Feature_selection), and is built into some machine learning approaches such as [lasso regression](https://en.wikipedia.org/wiki/Lasso_(statistics)).
```

Comment:
```suggestion
Dropping unnecessary variables is called [feature selection](https://en.wikipedia.org/wiki/Feature_selection), and is built into some machine learning approaches such as [lasso regression](https://en.wikipedia.org/wiki/Lasso_(statistics\)).
```
The final parenthesis was left off in Liascript, probably an artifact of needing an escape sequence?  We'll see

Reply:
Good catch!